include/isdimm_vpd_parser.hpp

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include "ipz_parser.hpp"
+#include "parser_interface.hpp"
+#include "types.hpp"
+
+namespace vpd
+{
+using ParserInterface = vpd::ParserInterface;
```

Comment:
ParserInterface , KeywordVpdMap, & isdimmVpdParser  are defined under vpd namespace. We dont need this redeclaration here, instead we can directly use.

Reply:
KeywordVpdMap is in types namespace, but may not be required removed.